cloud/server.go

```diff
@@ -516,7 +520,7 @@ func (s *Server) SSHClient(ctx context.Context) (*ssh.Client, int, error) {
 
 	// create a new client, add it to the pool
 	if s.sshClientConfig == nil {
-		err := s.createSSHClientConfig()
+		err := s.createSSHClientConfig(ctx)
```

Comment:
because the context here is already of clog.ContextWithServerID() type. 
All the methods calling SSHClient has their context of clog.ContextWithServerID() type.